src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -19,53 +19,52 @@
 public class AboutMeServlet extends HttpServlet{
 	private Datastore datastore;
 
- @Override
-  public void init() {
-  datastore = new Datastore();
- }
- 
+	@Override
+		public void init() {
```

Comment:
nit: FWIW, the Google style guide recommends using spaces, not tabs.

In general I'd recommend avoiding changing whitespace, because it makes it harder to spot your real changes.